x-pack/plugins/cases/server/client/cases/bulk_get.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Boom from '@hapi/boom';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { fold } from 'fp-ts/lib/Either';
+import { identity } from 'fp-ts/lib/function';
+import { pick, partition } from 'lodash';
+
+import type { SavedObjectError } from '@kbn/core-saved-objects-common';
+import { MAX_BULK_GET_CASES } from '../../../common/constants';
+import type { CasesBulkGetRequest, CasesBulkGetResponse } from '../../../common/api';
+import {
+  CasesBulkGetRequestRt,
+  CasesResponseRt,
+  excess,
+  throwErrors,
+  getTypeForCertainFieldsFromArray,
+  CaseResponseRt,
+} from '../../../common/api';
+import { getTypeProps } from '../../../common/api/runtime_types';
+import { createCaseError } from '../../common/error';
+import { asArray, flattenCaseSavedObject } from '../../common/utils';
+import { Operations } from '../../authorization';
+import type { CasesClientArgs } from '../types';
+import {
+  includeFieldsRequiredForAuthentication,
+  getAuthorizedAndUnauthorizedSavedObjects,
+} from '../../authorization/utils';
+import type { CaseSavedObject } from '../../common/types';
+
+type SOWithErrors = Array<CaseSavedObject & { error: SavedObjectError }>;
+
+/**
+ * Retrieves multiple cases by ids.
+ *
+ * @ignore
```

Comment:
nit: I think we can remove the `@ignore` tags, that was for the old way that we were generating the docs, I just haven't made the time to remove them all 😬